clang/test/CodeGenCXX/vtable-assume-load-nonzero-default-address-space.cpp

```diff
@@ -0,0 +1,288 @@
+// RUN: %clang_cc1 %s -triple=spirv64-unknown-unknown -fsycl-is-device -std=c++11 -emit-llvm -o %t.ll -O1 -disable-llvm-passes -fms-extensions -fstrict-vtable-pointers
+// FIXME: Assume load should not require -fstrict-vtable-pointers
+
+// RUN: FileCheck --check-prefix=CHECK1 --input-file=%t.ll %s
```

Comment:
I really don't understand why all these different CHECK lines are here... Should just be a single invocation (and could pipe directly to FileCheck). Looks like it dates all the way back to https://reviews.llvm.org/D11859.

Reply:
Does not need to be fixed as part of this change but would be great to consolidate those tests in a follow-up change.

Reply:
If you do not mind, I will take it up as a subsequent activity, as I'd have to figure out the rationale of the test to see how to collapse it. For now, it would be nice to not tickle the dragon too much, this has already grown quite a bit beyond its initial state:)

Reply:
Sounds good to me